index.html

```diff
@@ -19,6 +19,7 @@
   </header>
     <br><br>
   
+  <div id="content">
```

Comment:
- [ ] please try to fix the empty space on the right side of the website

![image](https://user-images.githubusercontent.com/89282221/185764966-6e5b537a-df46-4519-8199-88873a329f59.png)